### PR TITLE
Add Occlusion to Insights

### PR DESCRIPTION
**Pull request by edward-io**
image

### CODE REVIEW

File: captum/insights/config.py [hunk 1]
```diff
@@ -33,20 +43,47 @@ class StrEnumConfig(NamedTuple):
...
+    ),
+    FeatureAblation.get_name(): ConfigParameters(
+        params={"perturbations_per_eval": NumberConfig(value=1, limit=(1, 100)),}
+    ),
+    Occlusion.get_name(): ConfigParameters(
+        params={
+            "sliding_window_shapes": StrConfig(value=""),
+            "strides": StrConfig(value=""),
+        },
+        post_process={
```
Comment (J0Nreynolds) [inline captum/insights/config.py:84]: Why are these post-processing steps necessary?

Comment (edward-io) [reply captum/insights/config.py:84]: Occlusion takes in a tuple or an int e.g. `(3, 5, 5)`, so we need a way to convert a string representation into the appropriate format.

**Comment by edward-io** [comment]
Add fix to UI config parsing

**Comment by edward-io** [comment]
add perturbations_per_eval as a config option to occlusion

**Pull request closed without merging**